res/controllers/Akai-APC-MINI-scripts.js

```diff
@@ -0,0 +1,122 @@
+var APC = {};
+
+/*
+    AKAI APC Mini
+*/
+
+///////////////////////////////////////////////////////////////
+//                         FUNCTIONS                         //
+///////////////////////////////////////////////////////////////
+
+APC.init = function(id) { // called when the MIDI device is opened & set up
+    APC.id = id; // store the ID of this device for later use
+
+    for (var i = 1; i <= 64; i++) {
+        var g = "[Sampler" + i + "]";
+
+        engine.connectControl(g, "play", "APC.deckButtonPlay");
+        engine.connectControl(g, "track_loaded", "APC.deckTrackLoaded");
+        engine.connectControl(g, "eject", "APC.deckTrackEjected");
```

Comment:
Please don't use single-character variable names and don't use strings as callback arguments.

```suggestion
        var group = "[Sampler" + i + "]";

        engine.connectControl(group, "play", APC.deckButtonPlay);
        engine.connectControl(group, "track_loaded", APC.deckTrackLoaded);
        engine.connectControl(group, "eject", APC.deckTrackEjected);
```